care/utils/notification_handler.py

```diff
@@ -390,7 +403,10 @@ def generate(self):
                             json.dumps(
                                 {
                                     "external_id": str(notification_obj.external_id),
-                                    "title": self.message,
```

Comment:
Are you sure this wont break existing notifications?

Reply:
Yep, the service worker code has been updated to meet this change. This change was required because it was not possible before to send custom message without directly using the `send_webpush_user` method since there was no way to differentitate between a notification or a raw webpush message. This change allows us to do that using NotificationGenerator's `.generate`.

Service worker change: https://github.com/coronasafe/care_fe/pull/6224/files#diff-0aba05b7e68a2bb0e8cebbd6297fafc218eaeff08345a12bd379181c4a88e454L115

Service worker will auto update itself for every existing sessions on a page reload.